src/game/client/display.cpp

```diff
@@ -74,9 +78,110 @@ void Display::Reset()
  */
 void Display::Update()
 {
-#ifdef GAME_DLL
-    Call_Method<void, Display>(PICK_ADDRESS(0x00421870, 0x007BDB67), this);
-#endif
+    /*
```

Comment:
No need to leave this comment block here, I'll check the function over before I give it final approval.